### PR TITLE
Feature/setup associations

### PR DESCRIPTION
**Pull request by zhadier**
# The following changes were implemented in this Milestone :rice_cracker: 
- Established has-many relationship between Classroom and Student Classes
- Established many-many relationship between Perons and Book Classes using the intermediary Rental class.

## Thanks for taking out the time to review this project 🧨
It really is appreciated :) ❤️ 
- Note: I personally love optional suggestions so please do leave some 😄

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
```
Comment (EricMbouwe) [inline book.rb:1]: - [ ] A method for adding rentals should be present in this class to create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental

File: person.rb [hunk 2]
```diff
@@ -1,15 +1,16 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```
Comment (EricMbouwe) [inline person.rb:7]: - [ ] A person has many rentals, you should method for adding rentals. Let's fix that

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @zhadier,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
Hi @zhadier, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**